include/ccl_params.ini

```diff
@@ -13,11 +13,12 @@
 ;of filling these splines
 
 ;Scale factor spline
-A_SPLINE_DELTA=0.001
-A_SPLINE_NA=1000
-A_SPLINE_MIN=0.0003
+A_SPLINE_NA=250
```

Comment:
I was confused at first by there being two "NA"-related parameters. We have  A_SPLINE_NA and N_A. The latter is in also in the ccl_splines struct and it is used for the power spectrum spline. Perhaps we could rename N_A to A_SPLINE_NA_PK to make the difference more clear?

Reply:
Ok, I have renamed N_A to A_SPLINE_NA_PK (in include/ccl_params.h, include/ccl_params.ini, src/ccl_core.c, src/ccl_power.c and tests/ccl_test_cmblens.c).